libsolidity/Types.cpp

```diff
@@ -1012,6 +1018,17 @@ bool StructType::canLiveOutsideStorage() const
 	return true;
 }
 
+unsigned StructType::getSizeOnStack() const
+{
+	switch (location())
```

Comment:
I guess you plan to build more on this switch right? Cause it may be an overkil if it's just for these 2.